src/components/interface/Pagination/__tests__/__snapshots__/Pagination.test.js.snap

```diff
@@ -62,14 +62,15 @@ exports[`<Pagination /> should render pagination properly 1`] = `
       </a>
     </li>
     <li>
-      <a
+      <button
         aria-current="page"
         aria-label="page"
         class="fr-pagination__link"
-        title="page"
+        disabled=""
```

Comment:
`disabled` is risky https://a11y-manjula.netlify.app/#91

Reply:
update